velox/expression/Expr.cpp

```diff
@@ -1117,7 +1117,7 @@ void Expr::applyFunction(
     EvalCtx* context,
     VectorPtr* result) {
   stats_.numProcessedRows += rows.countSelected();
-  CpuWallTimer timer(stats_.timing);
+  auto timer = cpuWallTimer();
```

Comment:
probably `folly::doNotOptimizeAway(cpuWallTimer())`?

Reply:
Why do you think this would be needed? folly::doNotOptimizeAway is designed to be used in benchmarks to avoid eliminating truly useless variables.

Reply:
Since `timer` is not used, I felt there is a chance that the compiler could optimize the call away.